### DISCUSSION TITLE
Purported Evidence of the Reionization Era

### DISCUSSION THREAD

**Discussion by MJoseph315** [a-cosmology-group/acg#44] [Astrophysics and Physics]
Hey Everyone, I ended up coming across this article that purports to show evidence of the theoretical Era of Reionization in Big Bang Cosmology: (https://www.nasa.gov/feature/goddard/2023/nasa-s-webb-proves-galaxies-transformed-the-early-universe) Thoughts on this article?

**Comment by MJoseph315** [comment]
More information on the reported study here: (https://www.space.com/early-galaxies-transparent-universe-james-webb-space-telescope) quote from the article "the team saw that most of the gas in the universe was somewhere between opaque and transparent. But directly around the galaxies, everything was clear."

**Comment by RedshiftDrift** [comment]
From the articles, the Big Bang scenario is:
1. In the early universe, the gas between stars and galaxies was opaque – energetic starlight could not penetrate it.
2. After the Big Bang, the gas was hot and dense, then cooled, then again became hot and ionized – likely due to the formation of early stars in galaxies, and over millions of years, became transparent.
3. Scientists aimed Webb in the direction of a quasar.
4. Webb clearly show that these transparent regions are found around galaxies, we’ve also measured how large they are.
5. The galaxies are generally surrounded by transparent regions about 2 million light-years in radius.
6. Those early galaxies existed just 900 million years after the Big Bang.
7. Most of the gas in the universe was somewhere between opaque and transparent; directly around the galaxies, everything was clear.

**Thoughts**: use a [cosmology calculator on GitHub](https://a-cosmology-group.github.io/acg/code/marmet_l/cosmology-calculator.html) to find out what redshift corresponds to 900 Myr in LCDM.  This gives z = 6.2, a light travel time = 12.8 Gyr, and an angular scale 5.7 kpc/''.

### Interpreting this in a static cosmology

At z = 6.2, the light travel time is 26.1 Gly, and the angular scale is actually 38.7 kpc/'' (the galaxy is larger in size than what BB-cosmologists think by a factor of 38.7/5.7 = 6.8).  A point-by-point comparison gives:

1. 26 Gyr ago, the gas between stars and galaxies was not dense, it was ionized and just as transparent as it is today.
2. The gas is hot and ionized as far as we have seen – likely due to the stars in galaxies which keep the gas ionized, as it is today.
3. Scientists aimed Webb in the direction of a **quasar**, to get a bright object.
4. Webb clearly shows that these transparent regions are found around galaxies, we’ve also measured how large they are.
5. The galaxies are generally surrounded by transparent regions having a real size of 2 million $\times$ 6.8 = 13.6 million light-years in radius.  Note that the average separation between galaxies is about 10 million light-years (without considering they are grouped in clusters).
6. Those distant galaxies existed 26 billion years ago.
7. Most of the gas in the universe is transparent; directly around the galaxies all the way to the next galaxy, everything is clear.  The measured opacity is likely a result of light travelling through 6.8 times more gas than what Big Bang cosmologists think.

Because Big Bang cosmologists calculate distances and luminosity with LCDM equations, they "look at an illusion" of a very distorted universe.  Since there is no expansion, distant galaxies and the space between them are actually much larger that what Big Bang cosmologists think (even the luminosity distance is off by a factor of three, which produces an incorrect calculation of the ionization rate.)

Without a re-analysis of the data with a non-expanding cosmology, it is difficult to tell precisely where the results become incorrect in the LCDM framework.  However, the non-expanding model can approximately account for all of these observations, and these papers give no evidence of a reionization era.
